content/references/xrp-ledger-toml.md

```diff
@@ -280,16 +280,50 @@ Domain verification requires establishing a two-way link between the domain oper
 
     - In that `xrp-ledger.toml` file, provide a `[[VALIDATORS]]` entry with the validator's master public key in the `public_key` field.
 
-2. The validator claims ownership by the domain. The instructions for this step are out of scope for this specification.
+2. The validator claims ownership of the domain: 
 
-    <!-- TODO: Link documentation when it's available. -->
-    <!-- Aside: the old way of doing this was to set the `Domain` field of the
-         XRP Ledger account whose public key matched the validator's public key.
-         Since only the holder of the corresponding private key could send
-         transactions from this address, doing so effectively proved ownership
-         of the validator's private key.
-    -->
+    - Ensure that you have access to the validator-keys.json file that you created when first setting up your validator. If you have lost your keys or the keys have been compromised, please [revoke your keys](https://xrpl.org/run-rippled-as-a-validator.html#revoke-validator-keys) and generate new keys. 
```

Comment:
Links within the site should use the HTML filename (and anchor) only. Please remove the `https://xrpl.org/` prefix. This helps with the translated versions of the site and local testing. It also makes sure that they don't get styled as external links. (Also, if we move domains again, as we've done twice in the last 18 months, it saves us from doing a bunch of work. Hopefully we won't have to any time soon, but it's still good to be prepared.)